bentoml/_internal/frameworks/detectron.py

```diff
@@ -167,7 +167,7 @@ def save(
         "framework_name": "detectron2",
         "pip_dependencies": [
             f"detectron2=={_detectron2_version}",
-            f"torch_version=={_torch_version}",
```

Comment:
maybe we should also fix the detectron tests as well?

Reply:
the detectron tests failed randomly and I think it's because we fetch sample image from internet. Now I move that image to local